src/main/java/de/unistuttgart/t2/common/scaling/cpu/CPUUsageManager.java

```diff
@@ -0,0 +1,79 @@
+package de.unistuttgart.t2.common.scaling.cpu;
+
+import java.util.*;
+import java.util.concurrent.*;
+
+/**
+ * Manages how much CPU is consistently used as minimum.
+ *
+ * @author Leon Hofmeister
+ * @since 1.2.0
+ */
+public final class CPUUsageManager {
```

Comment:
maybe add private constructor to prevent instantiation

Reply:
Well, as it is now a bean, it'll be a public constructor.